js/ui/file_manager/ui.file_manager.item_list.js

```diff
@@ -80,7 +81,24 @@ class FileManagerItemListBase extends Widget {
 
     _getItemThumbnail(item) {
         const itemThumbnailGetter = this.option("getItemThumbnail");
-        return itemThumbnailGetter ? itemThumbnailGetter(item) : "";
+        return itemThumbnailGetter ? itemThumbnailGetter(item) : { thumbnail: "" };
+    }
+
+    _getItemThumbnailContainer(item) {
+        const { thumbnail, cssClass } = this._getItemThumbnail(item);
+
+        const $itemThumbnail = getImageContainer(thumbnail)
+            .addClass(this._getItemThumbnailCssClass());
+
+        if(cssClass) {
+            $itemThumbnail.addClass(cssClass);
```

Comment:
`return getImageContainer(thumbnail).addClass(this._getItemThumbnailCssClass()).addClass(cssClass || "");`